packages/edit-site/src/components/save-button/index.js

```diff
@@ -48,7 +53,7 @@ export default function SaveButton() {
 				isBusy={ isSaving }
 				onClick={ disabled ? undefined : open }
 			>
-				{ __( 'Update' ) }
+				{ __( 'Customize' ) }
```

Comment:
It'd make more sense for this to say "Update Design" or similar :)

Reply:
Agreed, updated 😄